repldex/discordbot/commands.py

```diff
@@ -119,7 +119,7 @@ async def link_source(message):
 
 
 @betterbot.command(name='entry', allowed=True)
-async def show_entry(message, search_query: str):
+async def show_entry(message, *, search_query: str):
```

Comment:
I'm pretty sure that won't work with repldex, this should be locally by an admeen

cc: @mat-1

Reply:
Tested*

Reply:
The reason it may not work is bc repldex doesn't use normal discord.ext but a custom thing made by mat

Reply:
I couldn't get a local bot working for the life of me, so ig an admin should test this.